tests/lib/metadata_linux_bridge_test.py

```diff
@@ -0,0 +1,366 @@
+#
+# Copyright (c) 2019 Red Hat, Inc.
```

Comment:
Everything looks fine to me. Just a nit: Should we use `Copyright (c) 2019 Red Hat, Inc.`?

Reply:
The header is identical to the others. Did I miss something?